database/index.js

```diff
@@ -0,0 +1,16 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  database: 'soundcloud',
+});
+
+connection.connect((err) => {
+  if (err) {
+    console.error(`error connecting: ${err.stack}`);
+    return;
+  } console.log('connected to database');
+});
+
+module.exports.connection = connection;
```

Comment:
Like above - unless you're exporting multiple things from this index file, I think you can just assign one thing to this export statement. It's a trifle more than anything else - especially if you intend to create other functions to export.